templates/anndata_conversion.ipynb

```diff
@@ -0,0 +1,784 @@
+{
```

Comment:
<p>Feel like sections 1.1 and 1.2 can get grouped down under like an "Optional Tutorial Section". It should be clear to the user what is a necessary cell to run (the ones above) and what is optional for them to understand the new data structure.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1079/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
That sounds like a good idea, I'll add an optional section and more the non conversion stuff over.